rhcos4/cpe/rhcos4-cpe-dictionary.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<cpe-list xmlns="http://cpe.mitre.org/dictionary/2.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://cpe.mitre.org/dictionary/2.0 http://cpe.mitre.org/files/cpe-dictionary_2.1.xsd">
+      <cpe-item name="cpe:/a:redhat:enterprise_linux_coreos:4">
+            <title xml:lang="en-us">Red Hat Enterprise Linux CoreOS release 4</title>
+            <!-- the check references an OVAL file that contains an inventory definition -->
+            <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="filename">installed_OS_is_rhcos4</check>
+      </cpe-item>
+      <cpe-item name="cpe:/a:machine">
+            <title xml:lang="en-us">Bare-metal or Virtual Machine</title>
+            <!-- the check references an OVAL file that contains an inventory definition -->
+            <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="filename">installed_env_is_a_machine</check>
+      </cpe-item>
+      <cpe-item name="cpe:/a:chrony">
+            <title xml:lang="en-us">Package chrony is installed</title>
+            <!-- the check references an OVAL file that contains an inventory definition -->
+            <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="filename">installed_env_has_chrony_package</check>
+      </cpe-item>
+      <cpe-item name="cpe:/a:gdm">
+            <title xml:lang="en-us">Package gdm is installed</title>
```

Comment:
I literally have zero idea what this file is for, but several of the items in this file will never be present on a RHCOS host, at least not in the OCP context (gdm, nss-pam-ldapd, yum, ...)

Reply:
@yuumasato @ggbecker any idea what's up with this file? I merely copied it from the ocp product.

Reply:
in ComplianceAsCode we use to limit applicability of certain checks using the `platform:` keyword within `rule.yml` files:

Check this excerpt from the developer_guide.adoc:
```
platform: Defines applicability of a rule. For example, if a rule is not applicable to containers, 
this should be set to machine, which means it will be evaluated only if the targeted scan 
environment is either bare-metal or virtual machine. Also, it can restrict applicability on higher 
software layers. By setting to shadow-utils, the rule will have its applicability restricted to only 
environments which have shadow-utils package installed. The available options can be found 
in the file <product>/cpe/<product>-cpe-dictionary.xml (e.g.: rhel8/cpe/rhel8-cpe-dictionary.xml).
 In order to support a new value, an OVAL check (of inventory class) must be created under 
shared/checks/oval/ and referenced in the dictionary file.
``